src/IssueViz/Commands/IssueVisualizationToolWindowCommand.cs

```diff
@@ -34,21 +35,38 @@ internal sealed class IssueVisualizationToolWindowCommand
     {
         public static readonly Guid CommandSet = Constants.CommandSetGuid;
         public const int CommandId = 0x0100;
+        public const int ErrorListCommandId = 0x0200;
 
         public static IssueVisualizationToolWindowCommand Instance { get; private set; }
 
         private readonly AsyncPackage package;
+        private readonly IVsMonitorSelection monitorSelection;
         private readonly ILogger logger;
+        private readonly uint uiContextCookie;
 
-        internal IssueVisualizationToolWindowCommand(AsyncPackage package, IMenuCommandService commandService, ILogger logger)
+        internal readonly OleMenuCommand ErrorListMenuItem;
+
+        internal IssueVisualizationToolWindowCommand(AsyncPackage package, IMenuCommandService commandService, IVsMonitorSelection monitorSelection, ILogger logger)
         {
             this.package = package ?? throw new ArgumentNullException(nameof(package));
+            this.monitorSelection = monitorSelection ?? throw new ArgumentNullException(nameof(monitorSelection));
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
-            commandService = commandService ?? throw new ArgumentNullException(nameof(commandService));
 
-            var menuCommandID = new CommandID(CommandSet, CommandId);
-            var menuItem = new MenuCommand(Execute, menuCommandID);
+            if (commandService == null)
+            {
+                throw new ArgumentNullException(nameof(commandService));
+            }
+
+            var menuCommandId = new CommandID(CommandSet, CommandId);
+            var menuItem = new MenuCommand(Execute, menuCommandId);
             commandService.AddCommand(menuItem);
+
+            menuCommandId = new CommandID(CommandSet, ErrorListCommandId);
+            ErrorListMenuItem = new OleMenuCommand(Execute, null, QueryStatus, menuCommandId);
```

Comment:
It also looks like a mistake that we are using the same `Execute` method for two seemly-different commands, even though it's not. A short comment would help e.g.
`We're showing the command in two places in the UI, but we only do a status check when it's called from the Error List context menu`.